public/api-documentation/swagger.json

```diff
@@ -401,6 +441,32 @@
         }
       }
     },
+    "SubmitSignature": {
+      "type": "object",
+      "properties": {
+        "answer": {
+          "type": "string"
+        },
+        "question_id": {
+          "type": "string"
+        },
+        "token": {
+          "type": "string"
+        },
+        "user_id": {
+          "description": "CDX user id",
+          "type": "string"
+        }
+      }
+    },
+    "SubmitTransaction": {
```

Comment:
If the primary noun is "submission", then it's trending to generalities. As a user, what does "submission" mean? I thought I already submitted the data to the e-Manifest system, what does it mean to create a submission again?